nest/src/config.ts

```diff
@@ -0,0 +1,23 @@
+export const config = () => ({
+    census: {
+        serviceID: process.env.CENSUS_SERVICE_ID,
+        environment: 'ps2',
+        worlds: process.env.CENSUS_SUBSCRIPTION_WORLDS.split(',') ?? ['all'],
+    },
+
+    rabbitmq: {
+        urls: [process.env.RMQ_URL ?? 'amqp://localhost:5672'],
```

Comment:
ENV names need to match: https://github.com/ps2alerts/aggregator/blob/revive/src/config/rabbitmq.ts

Unless you're willing to update all of the provisioning as well 😉

Reply:
👍